analysis/deep2_r23_o32/calibration_plots.py

```diff
@@ -9,48 +9,60 @@
 
 from .logging import log_stdout
 
-
-def lac_gpc_plots(fitspath, fitspath_ini, dataset, revised=False,
-                  individual=False, log=None):
+def lac_gpc_plots(fitspath, fitspath_ini, dataset, raw=False,
+                  apply_dust=False, revised=False, individual=False, log=NOne):
     """
-    Purpose
-    Call function for calculating and plotting data points based with the green_pea_calibration
-    and the local_analog_calibration.
-
-    Parameters
-    fitspath -> save location of the current run
-    fitspath_ini -> '/Users/reagenleimbach/Desktop/Zcalbase_gal/'
-    dataset  -> indicates the type of binning being used
-    revised  -> indicates that revised verification table is being used
-    individual -> used if individual detections from Zcalbase_gal are used
-
-    Outputs
-    pdf_files
+    Purpose:
+      Call function for calculating and plotting data points based with the green_pea_calibration
+      and the local_analog_calibration.
+
+    Parameters:
+      fitspath -> save location of the current run
+      fitspath_ini -> '/Users/reagenleimbach/Desktop/Zcalbase_gal/'
+      dataset  -> indicates the type of binning being used
+      revised  -> indicates that revised verification table is being used
+      individual -> used if individual detections from Zcalbase_gal are used
```

Comment:
Parameters
------------
param: fitspath: str. save location of the current run
param: fitspath_ini: str. path to location of project's parent folder
param: dataset: str. indicates the type of binning being used
param: revised: bool. indicates that revised verification table is being used
param: individual: bool. used if individual detections from Zcalbase_gal are used

Reply:
I created #64 for this issue. Will address outside this PR.